go.mod

```diff
@@ -1,15 +1,25 @@
 module github.com/stackrox/ossls
 
-go 1.14
+go 1.21
+
+toolchain go1.21.9
 
 require (
-	github.com/BurntSushi/toml v0.3.1
-	github.com/fatih/color v1.7.0
+	github.com/BurntSushi/toml v1.4.0
+	github.com/fatih/color v1.17.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/mattn/go-colorable v0.0.9 // indirect
-	github.com/mattn/go-isatty v0.0.8 // indirect
-	github.com/pkg/errors v0.8.1
-	github.com/spf13/cobra v0.0.4
+	github.com/pkg/errors v0.9.1
+	github.com/spf13/cobra v1.8.1
 	github.com/stretchr/testify v1.3.0
```

Comment:
I guess while you're at it, want to bump this to `v1.9.0`?

Reply:
Might as well :)